### PR TITLE
#108: add a static isSite function to Neodymium

### PR DESCRIPTION
**Pull request by jseng92**
please check and merge to develop

### CODE REVIEW

File: src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    {
+        Neodymium.configuration().setProperty("neodymium.url.site", "");
+        String expectedSite = "DE";
+        Assert.assertFalse(Neodymium.isSite(expectedSite));
+    }
+
+    @Test
+    public void varcharMatch() throws Exception
+    {
+
```
Comment (occupant23) [inline src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java]: remove newline

File: src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+import com.xceptance.neodymium.util.Neodymium;
+
+@RunWith(NeodymiumRunner.class)
+public class IsSiteTests
+{
+
+    @Test
+    public void happyPath() throws Exception
+    {
+
```
Comment (occupant23) [inline src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java]: remove newline

File: src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            {
+                "USA", "DE"
+            };
+        Assert.assertTrue(Neodymium.isSite(expectedSites));
+    }
+
+    @Test
+    public void reverseVarcharMatch() throws Exception
+    {
+
```
Comment (occupant23) [inline src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java]: remove newline

File: src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+            {
+                "DE", "USA"
+            };
+        Assert.assertTrue(Neodymium.isSite(expectedSites));
+    }
+
+    @Test
+    public void noMatchVarchar() throws Exception
+    {
+
```
Comment (occupant23) [inline src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java]: remove newline

File: src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+            {
+                "DE", "UK"
+            };
+        Assert.assertFalse(Neodymium.isSite(expectedSites));
+    }
+
+    @Test
+    public void nullVarchar() throws Exception
+    {
+
```
Comment (occupant23) [inline src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java]: remove newline

File: src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.xceptance.neodymium.NeodymiumRunner;
+import com.xceptance.neodymium.util.Neodymium;
+
+@RunWith(NeodymiumRunner.class)
+public class IsSiteTests
+{
+
```
Comment (occupant23) [inline src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java]: remove newline

File: src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+    {
+
+        Neodymium.configuration().setProperty("neodymium.url.site", null);
+        String[] expectedSites =
+            {
+                "DE", "UK"
+            };
+        Assert.assertFalse(Neodymium.isSite(expectedSites));
+    }
+}
```
Comment (occupant23) [inline src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java]: add another test with an the varchar containing a null value

**Comment by jseng92** [comment]
done

File: src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+        Neodymium.configuration().setProperty("neodymium.url.site", null);
+        String[] expectedSites =
+            {
+                "DE", "UK"
+            };
+        Assert.assertFalse(Neodymium.isSite(expectedSites));
+    }
+
+    @Test
+    public void ExpectedNullVarchar() throws Exception
```
Comment (occupant23) [inline src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java]: please use the standard Java naming scheme >> no capital letters for names of functions

File: src/main/java/com/xceptance/neodymium/util/Neodymium.java [hunk 9]
```diff
@@ -344,4 +344,26 @@ public static void timeout(long timeout)
...
+        for (int i = 0; i < sites.length; i++)
+        {
+            if (Neodymium.configuration().site().equals(sites[i]))
+            {
+                return true;
+            }
+        }
+        return false;
+    }
+
```
Comment (occupant23) [inline src/main/java/com/xceptance/neodymium/util/Neodymium.java]: remove newline

File: src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java (hunk 7 above)
Comment (occupant23) [reply src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java]: please change order of the ""DE, null in order to make contact wit the null value
